SConstruct

```diff
@@ -209,7 +211,11 @@ vars.AddVariables(
 
 ## Create a construction environment adding the build vars and
 ## propagating the user's external environment
-env = Environment(variables=vars, ENV=os.environ)
+env = Environment(variables=vars)
+
+## Load the PATH variable of the user environment,
+## to look for third-party library paths
+env.PrependENVPath('PATH', os.environ['PATH'])
 
```

Comment:
I would add something like:
```suggestion
try: 
    env.PrependENVPath("HDF5_DIR", os.environ["HDF5_DIR"])
except:
    Exception("HDF5_DIR must be defined")
```
Note that this defines the following entry in the `env` dictionary env["ENV"]["HDF5_DIR"], instead of env["HDF5_DIR"].

[Scons Documentation](https://www.scons.org/doc/production/HTML/scons-user.html#idm46358283013728), section 7.3

Reply:
I would leave it as it is. This way, it is possible to pass the HDF5 path via command line, like the other third-party libraries.